tests/clv/test_distributions.py

```diff
@@ -278,7 +278,9 @@ def test_pareto_nbd_sample_prior(
 
 
 class TestBetaGeoBetaBinom:
-    @pytest.mark.parametrize("batch_shape", [(), (5,)])
+    @pytest.mark.parametrize(
+        argnames="batch_shape", argvalues=[(), (5,)], ids=["scalar", "vector"]
+    )
     def test_logp_matches_lifetimes(self, batch_shape):
```

Comment:
Lines 298:
```
T = pm.draw(pm.DiscreteUniform.dist(1, 10, shape=batch_shape), random_seed=rng)
```
Setting `T` to a constant of 10 would be more realistic, but I'm not sure this will fix things.

This test was added out of a desire to compare with `lifetimes._likelihood`, but the ground truth values for when `lifetimes` was originally built & tested are in `test_logp_matches_excel`. If a vectorized likelihood matches those values, then this is an unnecessary test.

Reply:
I think it was worth to have as it helped me to get https://github.com/pymc-labs/pymc-marketing/pull/707/commits/da6568c846858723cc31ea931e51f3a6c0d7c375